tools/bumper/bumper.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+var Logger *log.Logger
```

Comment:
s/Logger/logger/ It can be confuse with a Logger struct

Reply:
Done